distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2045,19 +2045,18 @@ type Table
                 self.select_columns column . if_not_error <| lookup_table.select_columns [from_column, to_column] . if_not_error <|
```

Comment:
Please do this:
```suggestion
                selected_lookup_columns = lookup_table.select_columns [from_column, to_column]
                self.select_columns column . if_not_error <| selected_lookup_columns . if_not_error <|
```

then you can use `selected_lookup_columns` instead of running the select twice for no need.

Reply:
Done.